_studio/shared/src/libmfx_allocator_vaapi.cpp

```diff
@@ -303,7 +305,7 @@ mfxDefaultAllocatorVAAPI::AllocFramesHW(
                                       context_id,
                                       codedbuf_type,
                                       codedbuf_size,
-                                      1,
+                                      codedbuf_num,
```

Comment:
Why don't we use `vaCreateBuffer2` w/ proper width/height instead of this dirty hack w/ passing height as `num_elements` argument of `vaCreateBuffer`?

Reply:
I agree that it's more suitable here according to libva but driver team said that driver doesn't support vaCreateBuffer2 at all.